README.md

```diff
@@ -49,9 +49,40 @@ _tick_ is used for many industrial applications including:
 
 ## Installation
 
-### Requirements
+### Windows
 
-_tick_ currently works only on Linux/OSX systems and requires Python 3.4 or newer. Please have the required Python dependencies in your Python environment:
+  Requries:
+- python 3.4+
+- swigwin 3.0+
+- Microsoft Visual C++ (https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017)
+
+HowTO:
+
```

Comment:
Yes I didn't get this `vcvars.bat`, maybe should we be more explicit about how to find and use it. (it's almost the first time I open a Windows console so I come with all the newbies questions :)!)